rules/test-rule/rule.md

```diff
@@ -0,0 +1,54 @@
+---
+type: rule
+title: Induction task - Do you know this is a test rule?
+guid: df9552b7-1eff-4acc-9ae5-8e3db2d3cc82
+uri: test-rule
+created: 2023-01-23T15:23:10Z
+authors:
+  - title: Hugo Pernet
+    url: https://ssw.com.au/people/hugo-pernet
+---
+This is the introduction to my awesome rule.
+<!--endintro-->
+# Lists
```

Comment:
Hey @Freego1783 

please avoid making all headings h1 in the content...

https://webaim.org/projects/screenreadersurvey7/#heading

Reply:
Done - changed to h3